openbb_terminal/stocks/insider/sdk_helper.py

```diff
@@ -0,0 +1,229 @@
+    """GEt insider trades based on preset filter
+
+    Parameters
+    ----------
+    preset : str
+       Name of preset filter
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame of filtered insider data
+
+    Examples
+    --------
+    >>> from openbb_terminal.sdk import openbb
+
+    In order to filter, we pass one of the predefined .ini filters from OpenBBUserData/presets/stocks/insider
+    >>> filter = "Gold-Silver"
+    >>> insider_trades = openbb.stocks.ins.filter(filter)
+    """
+    url = openinsider_model.get_open_insider_link(preset)
+    return openinsider_model.get_open_insider_data(url, has_company_name=True)
+
+
+def lcb() -> pd.DataFrame:
+    """Get latest cluster buys
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame of latest insider trades
+
+    Examples
+    --------
+    >>> from openbb_terminal.sdk import openbb
+    >>> openbb.stocks.ins.lcb()
+    """
+    return openinsider_model.get_print_insider_data("lcb")
+
+
+def lpsb() -> pd.DataFrame:
+    """Get latest penny stock buys
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame of latest insider trades
+
+    Examples
+    --------
+    >>> from openbb_terminal.sdk import openbb
+    >>> openbb.stocks.ins.lpsb()
+    """
+    return openinsider_model.get_print_insider_data("lpsb")
+
+
+def lit() -> pd.DataFrame:
+    """Get latest insider trades
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame of latest insider trades
+
+    Examples
+    --------
+    >>> from openbb_terminal.sdk import openbb
+    >>> openbb.stocks.ins.lit()
+    """
+    return openinsider_model.get_print_insider_data("lit")
+
+
+def lip() -> pd.DataFrame:
+    """Get latest insider purchases
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame of latest insider trades
+
+    Examples
+    --------
+    >>> from openbb_terminal.sdk import openbb
+    >>> openbb.stocks.ins.lip()
+    """
+    return openinsider_model.get_print_insider_data("lip")
+
+
+def blip() -> pd.DataFrame:
+    """Get latest insider purchases > 25k
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame of latest insider trades
+
+    Examples
+    --------
+    >>> from openbb_terminal.sdk import openbb
+    >>> openbb.stocks.ins.blip()
+    """
+    return openinsider_model.get_print_insider_data("blip")
+
+
+def blop() -> pd.DataFrame:
+    """Get latest officer purchases > 25k
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame of latest insider trades
+
+    Examples
+    --------
+    >>> from openbb_terminal.sdk import openbb
+    >>> openbb.stocks.ins.blop()
+    """
+    return openinsider_model.get_print_insider_data("blop")
+
+
+def blcp() -> pd.DataFrame:
+    """Get latest CEO/CFO purchases > 25k
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame of latest insider trades
+
+    Examples
+    --------
+    >>> from openbb_terminal.sdk import openbb
+    >>> openbb.stocks.ins.bclp()
```

Comment:
@jmaslek I think you've reverted this :disappointed: 
Should be `blcp` instead of `bclp`

Reply:
rip.  my bad.  got trigger happy on the force push